### ISSUE TITLE
js3-mode doesn't love ES6

### ISSUE THREAD

**Issue by kozross** [thomblake/js3-mode#107]
I haven't found any way to not make js3-mode unhappy about certain features of ES6 code (namely default parameters). Is there an option I'm missing?

**Comment by osv** [comment]
and es7.. async/await, js2 mode dont love it too btw.

**Comment by thomblake** [comment]
Yep, anything that's not in ES5 will look like an error to js3-mode - it's based on an older version of Rhino and hasn't been kept up to date.  Unless someone jumps up and starts maintaining this more aggressively, you'd have more luck looking into js2-mode - it's very well-supported.

**Comment by osv** [comment]
js2 yes, but It still not support async await, especial using async in class. That is why I don't use classes :)
